src/OpenTelemetry.Extensions.Enrichment/README.md

```diff
@@ -6,6 +6,165 @@
 Contains OpenTelemetry .NET SDK telemetry enrichment framework
 which is used for enrichment of traces.
 
+## Introduction
+
+Telemetry enrichment attaches various types of information to traces.
+You can use the telemetry enrichment framework to attach any custom information
+that you would like to be present in all of your traces.
+
+With telemetry enrichment framework, you don't need to worry about attaching
+the information carefully to each telemetry object you touch.
+Instead, if you implement your enricher class, it  takes care of the details
+automatically. You simply register your class with the enrichment framework
+and the enrichment framework will make sure to call the `Enrich()` method of your
+class every time there is an `Activity` in your app.
+
 ## Traces
 
-TBD
+Currently this package supports trace enrichment only.
+
+### Steps to enable OpenTelemetry.Extensions.Enrichment
+
+You can view an example project using Enrichment at
+`/examples/enrichment/Examples.Enrichment`.
```

Comment:
nit: Make this into a link like `[Examples.Enrichment](../../examples/enrichment/Examples.Enrichment/Program.cs)`

Reply:
Updated